From 3b26fc167409a85652aa7577a6fc1914536c1df7 Mon Sep 17 00:00:00 2001
From: Santhosh Kumar <santhosh.kumar.selvaraj@qt.io>
Date: Fri, 19 Sep 2025 14:37:50 +0200
Subject: [PATCH 080/239] Mark security header for the qt labs settings

QUIP: 23
Task-number: QTBUG-136184
Pick-to: 6.8
Change-Id: I1667422991e98c7f3c0c72654dc7b4df2c23361b
Reviewed-by: Oliver Eftevaag <oliver.eftevaag@qt.io>
(cherry picked from commit e9f1f80dc9f9aa2d82f2e1195089daca416e9d27)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/labs/settings/qqmlsettings.cpp       | 1 +
 src/labs/settings/qqmlsettings_p.h       | 1 +
 src/labs/settings/qqmlsettingsglobal_p.h | 1 +
 3 files changed, 3 insertions(+)

diff --git a/src/labs/settings/qqmlsettings.cpp b/src/labs/settings/qqmlsettings.cpp
index 5d81bce9b6..60f1bbee64 100644
--- a/src/labs/settings/qqmlsettings.cpp
+++ b/src/labs/settings/qqmlsettings.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qqmlsettings_p.h"
 #include <qcoreevent.h>
diff --git a/src/labs/settings/qqmlsettings_p.h b/src/labs/settings/qqmlsettings_p.h
index a01d3e13be..82c1d3a0bd 100644
--- a/src/labs/settings/qqmlsettings_p.h
+++ b/src/labs/settings/qqmlsettings_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QQMLSETTINGS_P_H
 #define QQMLSETTINGS_P_H
diff --git a/src/labs/settings/qqmlsettingsglobal_p.h b/src/labs/settings/qqmlsettingsglobal_p.h
index 856821f48b..1a576dc06c 100644
--- a/src/labs/settings/qqmlsettingsglobal_p.h
+++ b/src/labs/settings/qqmlsettingsglobal_p.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2021 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:insignificant
 
 #ifndef QTLABSSETTINGSGLOBAL_P_H
 #define QTLABSSETTINGSGLOBAL_P_H
-- 
2.51.2

